tests/validators/union.test.ts

```diff
@@ -78,7 +78,15 @@ describe('UnionValidator', () => {
 		);
 	});
 
-	test('Clone', () => {
+	describe('Or', () => {
+		const orPredicate = s.string.or(s.number);
+
+		test('Or and union should be strict equal', () => {
+			expect(unionPredicate).toStrictEqual(orPredicate);
+		});
+	});
```

Comment:
This is not testing for the `.or` override in UnionValidator, it's testing BaseValidator's `.or`.

Reply:
Where you've found this line? It's from a old commit ig